editor/filesystem_dock.cpp

```diff
@@ -130,6 +140,7 @@ bool FileSystemDock::_create_tree(TreeItem *p_parent, EditorFileSystemDirectory
 			FileInfo fi;
 			fi.name = p_dir->get_file(i);
 			fi.type = p_dir->get_file_type(i);
+			fi.path = p_dir->get_path().plus_file(fi.name);
```

Comment:
From what I can tell, assigning this here is unnecessary(?). You only actually appear to use this value later on when attempting to get the filesystem path, but the way you use it - as `fi.path.get_base_dir()` - should be the same thing as `p_dir->get_path()` which is already assigned to the `lpath` variable declared earlier. And if you do end up needing the actual path, the `file_metadata` appears to already be doing that (and is set as the 0th slot metadata for the `file_item` `TreeItem`). Unless I'm misreading the code.